packages/flutter/ios/sentry_flutter/Sources/sentry_flutter/SentryFlutterPlugin.swift

```
@@ -646,6 +622,22 @@ public class SentryFlutterPlugin: NSObject, FlutterPlugin {
         }
         return nil
   }
+
+  @objc public class func addBreadcrumbAsBytes(_ breadcrumbBytes: NSData) {
+      guard let breadcrumbString = String(data: breadcrumbBytes as Data, encoding: .utf8),
+            let jsonData = breadcrumbString.data(using: .utf8),
+            let breadcrumbDict = try? JSONSerialization.jsonObject(with: jsonData) as? [String: Any] else {
```

Same as the other PR, maybe we can use NSData directly without the String pass in-between.

---

Also we soul catch errors and log instead of failing silently.